port_ocean/run.py

```diff
@@ -11,7 +11,7 @@
 from port_ocean.core.integrations.base import BaseIntegration
 from port_ocean.logger_setup import setup_logger
 from port_ocean.ocean import Ocean
-from port_ocean.port_defaults import initialize_defaults
```

Comment:
You should move this function out of the cli

The user dont have to install the cli

Reply:
please elaborate

Reply:
move back to the core